data/mods/Magiclysm/Spells/attunements/Shaman.json

```
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "butterfly_swarm",
+    "type": "SPELL",
+    "name": { "str": "Butterfly Swarm" },
+    "description": "Summon pack of butterflies with razor-sharp wings, to cut every living creature on your way.",
```

not sure why just living if its not bio
```suggestion
    "description": "Summon a pack of butterflies with razor-sharp wings, to cut every creature in your way.",
```